.github/workflows/pipelines-root.yml

```diff
@@ -131,14 +131,20 @@ jobs:
       - name: "[TerragruntExecute]: Authenticate with AWS and then Invoke Terragrunt"
         id: terragrunt
         if: ${{ steps.bootstrap.outputs.action == 'TERRAGRUNT_EXECUTE' }}
-        uses: ./pipelines-actions/.github/actions/pipelines-aws-execute
+        uses: ./pipelines-actions/.github/actions/pipelines-execute
+        env:
+          TERRAGRUNT_AUTH_PROVIDER_CMD: 'pipelines auth terragrunt-credentials --ci github-actions --cloud aws --wd .'
```

Comment:
@yhakbar I think this is a nice workaround where we can roll this out selectively without introducing an input variable to pipelines-execute - thoughts?

Reply:
It does feel a bit like spooky action at a distance, but the logs very clearly advertise that they're using it, so I'm not concerned about it.

I think the true next level approach is to have the `pipelines execute` invocation automatically set this as the default if it detects that it's running in the context of a GitHub Actions run (`CI` is set and `GITHUB_` env vars are populated, etc).

Overall, I think it's better than increasing the number of inputs on the action, though.